react-hook/cdn-implementation/index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Decimal Clock</title>
+    <link href="style.css" rel="stylesheet" />
+
+    <script
+      crossorigin
+      src="https://unpkg.com/react@17/umd/react.production.min.js"
+    ></script>
+    <script
+      crossorigin
+      src="https://unpkg.com/react-dom@17/umd/react-dom.production.min.js"
+    ></script>
+  </head>
+  <body>
+    <h2>Decimal Clock</h2>
+    <div id="root"></div>
+
+    <script>
+      const e = React.createElement;
+
+      const Digits = () => {
+        return [...Array(10)].map((_, i) =>
```

Comment:
`[...Array(10)]` can be declared as const outside component body to prevent updates